src/qml/README.md

```diff
@@ -0,0 +1,55 @@
+# Bitcoin Core QML GUI
+
+**WARNING: THIS IS EXPERIMENTAL, DO NOT USE BUILDS FROM THIS REPO FOR REAL TRANSACTIONS!**
+
+This directory contains the source code for an experimental Bitcoin Core graphical user interface (GUI) built using the [Qt Quick](https://doc.qt.io/qt-5/qtquick-index.html) framework.
+
+# Goals and Limitations
+
+The current Bitcoin Core GUI has gathered enough technical debt and hacked on features; it is time to begin anew.
+This project will start from a clean slate to produce a feature-rich GUI with intuitive user flows and first-class design.
+
+The primary goals of the project can be summed up as follows:
+
+- Implement UX/UI best-practices as documented in the [Bitcoin Design Guide](https://bitcoin.design/guide/)
+- Engage with the Bitcoin Design community to implement well-designed features
+- Work alongside the Bitcoin Design community to develop an aesthetic GUI
+- Develop a mobile-optimized GUI
+
+It is important that we stay as conflict-free as possible with the Core repo.
+As such, this project will aim to make very few changes outside of the `qml` directory.
+Pull requests must be focused on developing the GUI itself, adding build support,
+or improving relevant documentation.
+
+Note that this project will **not** accept pull requests making any significant changes unrelated to the GUI.
+
+# Development Process
+
+This repo is synced with the [Core repo](https://github.com/bitcoin/bitcoin) on a weekly basis, or as needed to resolve conflicts.
```

Comment:
Please do not contract "Bitcoin Core" to "Core".

Reply:
done in https://github.com/bitcoin-core/gui-qml/commit/605842aef74c8c9f79e1fa631b496a991a78a30f